experimental/traffic-portal/build/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "traffic-portal-server",
+  "version": "1.0.0",
+  "description": "Node dependencies used for running traffic-portal",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/apache/trafficcontrol.git"
+	},
+	"bugs": {
+		"email": "dev@trafficcontrol.apache.org",
+		"url": "https://github.com/apache/trafficcontrol/issues"
+	},
+	"author": {
+		"email": "dev@trafficcontrol.apache.org",
+		"name": "Apache Software Foundation",
+		"url": "https://trafficcontrol.apache.org"
+	},
+	"engines": {
+		"node": ">=16"
+	},
+	"private": true,
+	"license": "Apache-2.0",
+  "dependencies": {
+    "pm2": "^5.2.2"
+  }
+}
```

Comment:
Why is this different than `experimental/traffic-portal/package.json`? Can we not use that one to make the RPM spec?

Reply:
Both exist in the source RPM, but _only_ this one exists in the prod RPM. Including the entire project's node_modules increases uncompressed install size by ~20MB (and compressed size by multiple MB but I forget that number). If we were to use apache2/nginx as the webserver instead then we wouldn't need this.

Reply:
Does that mean the RPMs being output can't run the server and require an external HTTP server? Only the SRPMs actually contain the server code?

Reply:
No. It is entirely possible to run the installed RPM using `node ./server/main.js` instead. `pm2` provides additional functionality such as auto restart, pid files, etc. It's also what TPv1 uses which makes reusing the same box for both TPv1/v2 simpler.

Reply:
Oh I misunderstood what you said above. Yeah, alright, sounds fine.